app/src/main/java/org/schabi/newpipe/local/playlist/LocalPlaylistFragment.java

```diff
@@ -472,13 +472,59 @@ public void removeWatchedStreams(final boolean removePartiallyWatched) {
                         "Removing watched videos, partially watched=" + removePartiallyWatched))));
     }
 
+    /**
+     * If the current thumbnail has changed but the video is still the same, this method will update
+     * the playlist thumbnail accordingly.
+     *
+     * @param currentItems List of items currently in the playlist
+     * @param updatedItems List of items that will replace the current items in the playlist
+     */
+    private void updateCurrentThumbnail(@NonNull final List<LocalItem> currentItems,
+                                        @NonNull final List<PlaylistStreamEntry> updatedItems) {
+
+        final String playlistThumbnail = playlistManager.getPlaylistThumbnail(playlistId);
+        // StreamId of the video that provides the thumbnail for the playlist (stays at index -1 if
+        // the video is no longer in the playlist).
+        long streamId = -1;
```

Comment:
Thx for your comment. Actually I was thinking about something like this:

```suggestion
        // StreamId of the video that provides the thumbnail for the playlist (stays at index -1 if
        // the video is no longer in the playlist).
        private static final int VIDEO_ID_NO_LONGER_IN_PLAYLIST = -1;
        
        long streamId = VIDEO_ID_NO_LONGER_IN_PLAYLIST;
```

Reply:
I was actually thinking of adding another constant, but since the "-1" would just be a helper index and could be any random negative number, I thought a small comment would suffice.
But I understand your perspective as well and want to align with NewPipes style, so I'll implement it now. 😄